jbpm-designer-backend/src/test/java/org/jbpm/designer/bpmn2/impl/Bpmn2UnmarshallingTest.java

```diff
@@ -1533,6 +1533,14 @@ public void testMITaskProperties() throws Exception {
     }
 
     @Test
+    public void testNoMIOnEmbeddedSubprocess() throws Exception {
+        Definitions definitions = loader.loadProcessFromJson("simpleEmbeddedSubprocess.json");
+        Process process = getRootProcess(definitions);
+        SubProcess subprocess = (SubProcess) getFlowElement(process.getFlowElements(), "abc");
+        assertNull(subprocess.getLoopCharacteristics().getId());
```

Comment:
I think with the eclipse bpmn2 api the loop characteristics will currently not be null. Could we please +1 this to get it in as its urgent and test works and please add a commit that does what you would like to do after? WDYT?